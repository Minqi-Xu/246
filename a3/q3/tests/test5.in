add body
add header
value CS246
parent
add text
value Hello
id SAY
parent
add searchtest
id one
parent
add cut
id CT
parent
add searchtest
id two
parent
list
find searchtest
value FIRSTOCCUR
up
print
find body
find cut
value cutting
cut
up
find body
up
print
quit
