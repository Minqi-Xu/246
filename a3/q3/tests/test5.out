[Command?]
<body> is now the current tag.
[Command?]
<header> is now the current tag.
[Command?]
<header> value set to 'CS246'.
[Command?]
<body> is now the current tag.
[Command?]
<text> is now the current tag.
[Command?]
<text> value set to 'Hello'.
[Command?]
<text> id set to 'SAY'.
[Command?]
<body> is now the current tag.
[Command?]
<searchtest> is now the current tag.
[Command?]
<searchtest> id set to 'one'.
[Command?]
<body> is now the current tag.
[Command?]
<cut> is now the current tag.
[Command?]
<cut> id set to 'CT'.
[Command?]
<body> is now the current tag.
[Command?]
<searchtest> is now the current tag.
[Command?]
<searchtest> id set to 'two'.
[Command?]
<body> is now the current tag.
[Command?]
<body> has the following children:
- <header>
- <text>
- <searchtest>
- <cut>
- <searchtest>
[Command?]
<searchtest> is now the current tag.
[Command?]
<searchtest> value set to 'FIRSTOCCUR'.
[Command?]
<document> is now the current tag.
[Command?]
<document>
  <body>
    <header value='CS246'>
    </header>
    <text id='SAY' value='Hello'>
    </text>
    <searchtest id='one' value='FIRSTOCCUR'>
    </searchtest>
    <cut id='CT'>
    </cut>
    <searchtest id='two'>
    </searchtest>
  </body>
</document>
[Command?]
<body> is now the current tag.
[Command?]
<cut> is now the current tag.
[Command?]
<cut> value set to 'cutting'.
[Command?]
<cut> is now in the clipboard.
<body> is now the current tag.
[Command?]
<document> is now the current tag.
[Command?]
<body> is now the current tag.
[Command?]
<document> is now the current tag.
[Command?]
<document>
  <body>
    <header value='CS246'>
    </header>
    <text id='SAY' value='Hello'>
    </text>
    <searchtest id='one' value='FIRSTOCCUR'>
    </searchtest>
    <searchtest id='two'>
    </searchtest>
  </body>
</document>
[Command?]
